fiftyone/core/odm/dataset.py

```diff
@@ -250,14 +251,18 @@ class ColorScheme(EmbeddedDocument):
             -   ``valueColors`` (optional): a list of dicts specifying colors
                 to use for individual values of this field
             -   ``maskTargetsColors`` (optional): a list of dicts specifying
-                index and color for 2D masks
+                index and color for 2D masks in the same format as described
+                below for default mask targets
         default_mask_targets_colors (None): a list of dicts with the following
             keys specifying index and color for 2D masks of the dataset. If a
             field does not have field specific mask targets colors, this list
             will be used:
 
-            -   ``intTarget``: integer target value
+            -   ``intTarget``: an integer target value
             -   ``color``: a color string
+
+            Note that the pixel value ``0`` is a reserved "background" class
```

Comment:
@lanzhenw if this is not already the case, can you update the frontend so that if maskTargets are provided that contain `target=0`, it will simply ignore the value rather than crashing?

Here's how this is documented for `Segmentation` fields:
<img width="648" alt="Screenshot 2024-09-11 at 11 26 35 PM" src="https://github.com/user-attachments/assets/d5f4f7fb-9520-47c6-b286-d9d319ad01bc">

Note that it doesn't say that `target=0` is not allowed. It just says that this is a special value that is always rendered as invisible in the App, which means that if the user provided `0: "background"` in their mask targets, then they won't see `background` in the tooltip when hovering over 0 pixels in the App.

The consistent behavior for color schemes would be to silently ignore a color if it is provided for pixel value 0.

Reply:
It's currently not the case. App crashes when intTarget 0 color is set. Working on it.